### DISCUSSION TITLE
Add man page to jc package

### DISCUSSION THREAD

**Discussion by kellyjonbrazil** [Homebrew/discussions#1221] [Everyday usage]
It's been a while since I originally created the `jc` package and it has had several updates since then. I just released v1.15.0 and I'd like to include the man page, if possible.

The man page can be downloaded from https://raw.githubusercontent.com/kellyjonbrazil/jc/master/man/jc.1.gz

Is there a best practice for this? BTW, `jc` is a python cli/library, so not sure if that makes a difference.

**Comment by carlocab** [comment]
I've updated the formula to also install the manpage here: https://github.com/Homebrew/homebrew-core/pull/74783

It would help for the next release if the manpage were just part of the PyPI tarball. Declaring the manpage as a resource (as I have in the 1.15 update) makes it harder to use Homebrew tooling to update the formula's resources automatically. (`brew update-python-resources` doesn't like it when there are non-PyPI resources in the formula.)

**Reply by carlocab** [reply]
Actually, I just realised that having an external manpage not only breaks `brew` tooling for updating the formula, it also breaks the `virtualenv_install_with_resources` method, which expects all the formula's resources to be tarballs that can be installed with `pip`.

I can change up the `install` method to get around this, but this adds way too much complexity to the maintenance of the formula for my taste.

**Reply by kellyjonbrazil** [reply]
No worries - I’ll look into adding the man page to the next pypi package. Thanks for looking into this!

**Reply by kellyjonbrazil** [reply]
I just released v1.15.1 on pypi and added a `/jc/man` folder in the package with the man file. Does that make it easier to add to the homebrew formula?

**Reply by carlocab** [reply]
Much. Opened https://github.com/Homebrew/homebrew-core/pull/75132 to update the formula and install the manpage.
